clang/include/clang/Frontend/MultiplexConsumer.h

```diff
@@ -53,6 +53,7 @@ class MultiplexConsumer : public SemaConsumer {
 public:
   // Takes ownership of the pointers in C.
   MultiplexConsumer(std::vector<std::unique_ptr<ASTConsumer>> C);
+  MultiplexConsumer(std::unique_ptr<ASTConsumer> C);
```

Comment:
This feels unnecessary because you should be able to do `MultiplexConsumer M({std::move(ptr)});` right?

Reply:
Surprisingly we can’t do this unfortunately https://godbolt.org/z/1e5oKjdhh